iphone/Maps/Images.xcassets/About/ic_social_media_matrix.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "ic_social_media_matrix.pdf",
```

Comment:
In some cases it is safer to use pdfs for the image assets in iOS. Let's use the same format as for the others social media icons.

Reply:
@kirylkaveryn, hello! I've replaced the file using PDF. Could you please explain in which cases we use PDFs? I see from [Apple's Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/icons) that we can use both PDF and SVG, but there is no information about compatibility or other details.

Reply:
Usually, ios draws svgs and pdfs well, but sometimes (not usually) it fails to draw the complex svg gradients with paths.
In this particular case, it is just to use a consistent format for the similar.